$NetBSD: patch-configure,v 1.1 2018/06/04 05:09:33 dbj Exp $

There is a typo in the gexiv2-0.10.8 configure.ac and resulting configure

--- configure.orig	2018-02-05 21:57:53.000000000 +0000
+++ configure
@@ -18635,7 +18635,7 @@ case "$target_or_host" in
 esac
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $platform_darwin" >&5
 $as_echo "$platform_darwin" >&6; }
- if test "$platform_win32" = "yes"; then
+ if test "$platform_darwin" = "yes"; then
   PLATFORM_DARWIN_TRUE=
   PLATFORM_DARWIN_FALSE='#'
 else
